ui/e2e/config.js

```diff
@@ -17,10 +17,10 @@ export const KEPLR_CONFIG = {
 };
 
 export const MM_CONFIG = {
-  id: "nkbihfbeogaeaoehlefnkodbefgpgknn",
-  ver: "9.1.1_0",
+  fileId: "kbncgnhjhkdmacanlpmplcbbncoblkfo", // zip name id is static, where ext id is dynamic and now stored in jest globals config
```

Comment:
This looks odd the id should not change. Can we get on a call about this? Are you sure you have used the correct zips from https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn

Reply:
You were right @ryardley , changing id was unexpected. I updated PR description (and the whole PR) that describes what needs to be done to have non-changing id.